programs/wen_new_standard/src/instructions/group/create.rs

```diff
@@ -72,6 +71,9 @@ pub struct CreateGroupAccount<'info> {
     )]
     pub mint_token_account: Box<InterfaceAccount<'info, TokenAccount>>,
     #[account(
+        init_if_needed,
```

Comment:
moving init to separate instruction, see PR https://github.com/wen-community/wen-new-standard/pull/32